packages/teams-js/src/internal/childCommunication.ts

```diff
@@ -157,17 +165,32 @@ function sendChildMessageToParent(
     message.args,
     true, // Tags message as proxied from child
   );
+
+  // Copy variable to new pointer
+  const requestChildWindowOrigin = ChildCommunication.origin;
   setCallbackForRequest(request.uuid, (...args: any[]): void => {
-    if (ChildCommunication.window) {
-      const isPartialResponse = args.pop();
+    if (!ChildCommunication.window) {
+      return;
+    }
+
+    if (
+      !getCurrentFeatureFlagsState().disableEnforceOriginMatchForChildResponses &&
+      requestChildWindowOrigin !== ChildCommunication.origin
```

Comment:
Is this right? Wouldn't we want requestChildWindowOrigin to match ChildCommunication.origin when enforceOriginMatchForChildResponses is enabled?